CONTRIBUTING.md

```diff
@@ -73,6 +73,14 @@ Don't:
 If no reference doc exists yet, then you can use the `description` as a stopgap for explaining
 both why the audit is important and how to fix it.
 
+## Updating sample artifacts and LHR JSON
+
+```
+yarn run update:sample-artifacts # update all artifacts
+yarn run update:sample-artifacts ScriptElements # update just one artifact
```

Comment:
should we note that they'll need to revert the devtoolslog and trace files if they want only this artifact?

Reply:
Added, feel free to reword.